src/parser/jobs/brd/modules/MultiHitSkills.ts

```diff
@@ -4,20 +4,24 @@ import DISPLAY_ORDER from './DISPLAY_ORDER'
 
 export default class MultiHitSkills extends AoEUsages {
```

Comment:
We're trying to move away from `export default` so this should just be `export class` (and update the index to import it as `import {MultiHitSkills}`)